ipmof_interpenetration.py

```diff
@@ -23,7 +23,9 @@
     mof_list = get_mof_list(mol2_list, force_field)
 else:
     # Create MOF list by reading mol2 files from a directory
-    mof_list = get_mof_file_list(sim_dir['mof_dir'], 'cif', force_field)
+    mof_path_list = os.listdir(sim_dir['mof_dir'])
+    mof_path_list = [os.path.join(sim_dir['mof_dir'], path) for path in mof_path_list]
+    mof_list = get_mof_list(mof_path_list, force_field)
```

Comment:
I will look into that to see if there are any differences. If not will create a common method in later PRs.